app/helpers/claims_helper.rb

```diff
@@ -38,6 +38,12 @@ def identity_answers(claim)
     ]
   end
 
+  def subject_list(subjects)
+    connector = " or "
+    subjects.map! { |subject| I18n.t("tslr.questions.eligible_subjects.#{subject}") }
```

Comment:
Is this mutating the original value passed in to the helper? I know this is a smell in other languages, but not sure how Ruby handles it

Reply:
Good call. That's a bad habit from a while back. Usually Rubocop picks that up